deb/index_files.go

```diff
@@ -107,6 +123,27 @@ func (file *indexFile) Finalize(signer pgp.Signer) error {
 			file.parent.renameMap[filepath.Join(file.parent.basePath, file.relativePath+file.parent.suffix+ext)] =
 				filepath.Join(file.parent.basePath, file.relativePath+ext)
 		}
+
+		if file.acquireByHash {
+			sums := file.parent.generatedFiles[file.relativePath+ext]
+
+			err = packageIndexByHash(file, ext, "SHA512", sums.SHA512)
+			if err != nil {
+				fmt.Printf("%s\n", err)
```

Comment:
We should return an error here? Unless we want to ignore some errors (I would prefer error to be returned)

Reply:
Agree. Changed.